requirements.txt

```diff
@@ -1,5 +1,5 @@
 git-lfs
-natcap.invest
+natcap.invest@git+https://github.com/natcap/invest@release/3.16.0
```

Comment:
After this PR is merged, and the invest release is done, we can then revert this to just `natcap.invest`.